### ISSUE TITLE
Fails on regular expressions

### ISSUE THREAD

**Issue by vitaly-t** [jsdoc3/jsdoc#1053]
I'm getting the following error:
```
ERROR: Unable to parse ..\lib\formatting.js: Line 82: Invalid regular expression
```
line 82:
```javascript
var svn = name.replace(/^\$[{(<[/]|[\s})>\]/]/g, '');
```
That's a valid regular expression that passes numerous tests.

I'm using the 3.3.2 official release version.

I'm surprised it is even trying to parse regular expressions in my code. Why is jsDoc doing it?

**Comment by hegemonic** [comment]
On the releases/3.3 branch, I've updated to a newer version of Esprima, which fixes the issue. The fix will be included in JSDoc 3.3.3.

**Comment by vitaly-t** [comment]
Any idea when 3.3.3 makes the release? I need this fix released very much.

Cheers!

**Comment by hegemonic** [comment]
@vitaly-t I just released JSDoc 3.3.3. Sorry for the delay.
